zebra-utils/src/bin/zebra-tip-height/main.rs

```diff
@@ -0,0 +1,54 @@
+//! Prints the chain tip height stored in Zebra's state.
+//!
+//! For usage please refer to the program help: `zebra-tip-height --help`
+
+use color_eyre::eyre::{eyre, Result};
+use structopt::StructOpt;
+use tower::util::BoxService;
+
+use zebra_chain::chain_tip::ChainTip;
+use zebra_state::LatestChainTip;
+use zebra_utils::init_tracing;
+
+mod args;
+use self::args::Args;
+
+/// Type alias for a boxed state service.
+type BoxStateService =
+    BoxService<zebra_state::Request, zebra_state::Response, zebra_state::BoxError>;
```

Comment:
Build time: this type alias is effectively unused. If we drop the service without declaring its type, we can avoid a tower dependency.

Reply:
Fixed! I was hesitant to not return the state service in case it needs to outlive the chain tip receiver, but it seems that's not necessary 🎉